.github/workflows/pull_request.yaml

```diff
@@ -0,0 +1,41 @@
+name: pull_request
+
+on:
+  push:
+    branches:
+    - 'master'
+  pull_request: {}
+
+jobs:
+  test:
+    name: Tests
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Cancel Previous Runs
+      uses: styfle/cancel-workflow-action@0.4.0
+      with:
+        access_token: ${{ github.token }}
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Set up Go 1.14
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.14
+    - uses: actions/cache@v1
+      with:
+        path: ~/go/pkg/mod
+        key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go
+    - uses: azure/setup-kubectl@v1
+      with:
+        version: 'v1.18.0'
+    - uses: engineerd/setup-kind@v0.3.0
+    - name: Build
+      run: |
+          go build ./...
+    - name: Run tests
+      run: |
+        go get -v github.com/onsi/ginkgo/ginkgo@v1.12.0 && export PATH=$PATH:$(go env GOPATH)/bin/
+        ginkgo  -r   -p   -failFast   -randomizeSuites   -randomizeAllSpecs   -skipPackage=./installutils/kubeinstall,./debugutils/test
```

Comment:
we don't need the skip package flag anymore

Reply:
We do, actually - it doesn't pass when I remove it.

Reply:
Though I haven't tried it again since fully upgrading go-utils, so let's see what happens with this next run.